kubernetes/README.md

```diff
@@ -55,4 +55,12 @@ to create the *secret* containing the required credentials in k8s.
 - Reload metadata
   - run `hasura-dev metadata reload --endpoint "http://localhost:7070" --admin-secret HASURA_GRAPHQL_ADMIN_SECRET`
 
+## How to set environment variables
 
+1. The helm chart first takes all the variables from [values.yaml](./polkassembly/values.yaml).
+1. Then if the helm program is called with the `--values=` flag it takes all the variables from this file and applies it (possibly overwirting variables from the previous `values.yaml`).
```

Comment:
```suggestion
1. Then if the helm program is called with the `--values=` flag it takes all the variables from this file and applies it. In our case, it takes the files [values-dashboards-cluster-1.yaml (for test)](./polkassembly/values-dashboards-cluster-1.yaml) or [values-parity-prod.yaml (for prod)](./polkassembly/values-parity-prod.yaml). It may overwrites variables that were set in `values.yaml`.
```